src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/EmailConfigurationType.php

```diff
@@ -102,6 +104,27 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('log_emails', SwitchType::class, [
                 'label' => $this->trans('Log Emails', 'Admin.Advparameters.Feature'),
             ])
+            ->add('dkim_enable', SwitchType::class, [
+                'label' => $this->trans('Enable DKIM', 'Admin.Advparameters.Feature'),
```

Comment:
I think it would be better to have "DKIM" as a label and replace 'yes/no' with 'Enabled/Disabled' in the toggle. But I couldn't find the Yes/No wording to modify it.